frontend/src/features/login/SsoHoldingPage.tsx

```diff
@@ -0,0 +1,59 @@
+import { useEffect, useMemo } from 'react'
+import { useTranslation } from 'react-i18next'
+import { useSearchParams } from 'react-router-dom'
+import { useLocalStorage } from 'react-use'
+import { Stack } from '@chakra-ui/react'
+import { StatusCodes } from 'http-status-codes'
+
+import { LOGGED_IN_KEY } from '~constants/localStorage'
+import { DASHBOARD_ROUTE, LOGIN_ROUTE } from '~constants/routes'
+import { useToast } from '~hooks/useToast'
+import Spinner from '~components/Spinner'
+
+export const SsoHoldingPage = () => {
+  const { t } = useTranslation()
+
+  const [, setIsAuthenticated] = useLocalStorage<boolean>(LOGGED_IN_KEY)
+
+  const [params] = useSearchParams()
+  const toast = useToast({ isClosable: true, status: 'danger' })
+
+  const statusCode = params.get('status')
```

Comment:
noting this down as part of my thought process for clarity:

we seem to be using the status parameter as a message-passing mechanism:
- 200: https://uat.form.gov.sg/login/sso?status=200 -> DASHBOARD_ROUTE
- 401: https://uat.form.gov.sg/login/sso?status=401 (Unauthorized) -> LOGIN_ROUTE
- 403: https://uat.form.gov.sg/login/sso?status=403 (Forbidden) -> LOGIN_ROUTE